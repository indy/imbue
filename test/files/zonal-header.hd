---
{
"names": ["foo", "bar", "baz"]
}
---
#hello
<% names.forEach(function(name) { %>* <%= name %>
<% })%>
#world  
<% zonal.names.forEach(function(name) { %>* <%= name %>
<% })%>