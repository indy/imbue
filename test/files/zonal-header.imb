---
"names": ["foo", "bar", "baz"],
"title": "hello",
"subtitle": "world"
---
#{{= page.title }}
{{=: page.names | prefixWith:'* '}}
#{{= page.subtitle }}
{{=: zonal.names | prefixWith:'* '}}
