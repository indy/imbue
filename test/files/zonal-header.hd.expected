<h1 id="hello">hello</h1>

<ul>
<li>foo</li>
<li>bar</li>
<li>baz</li>
</ul>

<h1 id="world">world</h1>

<ul>
<li>baq</li>
<li>baqq</li>
<li>baqqq</li>
</ul>